src/OmniSharp.Roslyn.CSharp/Services/Buffer/ChangeBufferService.cs

```diff
@@ -0,0 +1,26 @@
+using System.Composition;
+﻿using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using OmniSharp.Mef;
+using OmniSharp.Models;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Buffer
+{
+    [OmniSharpHandler(typeof(RequestHandler<ChangeBufferRequest, object>), LanguageNames.CSharp)]
```

Comment:
I'm not saying it can't be done, just that it would look weird.

In my head, every handler should implement IHandler and have a Handle method.